hls4ml/model/profiling.py

```diff
@@ -159,49 +159,48 @@ def types_histogram(data, fmt='longform'):
 types_plots = {'boxplot' : types_boxplot,
                'histogram' : types_histogram}
 
-def ap_fixed_WIF(dtype):
+def ap_fixed_WIFS(dtype):
```

Comment:
does this support `ap_[u]int` datatypes? Seeing as they seem to be supported in [`config.py`](https://github.com/fastmachinelearning/hls4ml/pull/405/files#diff-8ec43234900e263f95104d90f7969c22a9d314817d20ce4d1f23f1a4354fc91aR63) maybe we should support them here too?

Reply:
Yeah even though the function is called `ap_fixed_WIFS`, it does also work for the int types since the `IntegerPrecisionType` and `FixedPrecisionType` have the same members. [Here's a demo](https://gist.github.com/thesps/bd862e2bb1e91710fb2809676e2fff2e) evaluated on this PR branch. It also runs on master, but the box for the unsigned case is displayed wrongly as for `ap_fixed`.